app/models/partners/family.rb

```diff
@@ -58,6 +62,10 @@ def create_authorized
       )
     end
 
+    def archive_children
+      children.each { |c| c.update(archived: true) }
```

Comment:
You can just do `children.update_all(archived: true, updated_at: Time.zone.now)`

Reply:
Nice! I originally had it as `children.update_all(archived: true)` but rubocop recommended a change since update_all bypasses the validations

Reply:
I am seeing rubocop fail in the pipeline again, would you recommend using something like `rubocop:disable Rails::SkipsModelValidations` to skip the validations on this one line?

Reply:
In this case `Child` doesn't have any validations, so sure.